website/blog/2022-07-28-srb-tapioca.md

```diff
@@ -0,0 +1,266 @@
+---
+id: srb-tapioca
+title: Tapioca is the recommended way to generate RBIs for Sorbet
+---
+
+Today we're excited to announce that the Sorbet team officially recommends using
+Shopify's [Tapioca](https://github.com/Shopify/tapioca) gem for RBI generation
+in projects that use Sorbet. Tapioca takes the place of the `srb rbi` family of
+commands, which as of today are officially in maintenance mode. Sorbet itself
+remains in active development.
+
+<!--truncate-->
+
+![sorbet plus tapioca](/img/sorbet-plus-tapioca.svg)
+
+To elaborate, here's what's happening:
+
+- The [`tapioca`](https://github.com/Shopify/tapioca) gem is now the recommended
+  approach to [RBI generation](/docs/rbi) in projects that use Sorbet. This
+  recommendation is reflected in the Sorbet docs. Tapioca replaces (and improves
+  on!) functionality currently provided by the `srb rbi` command. Tapioca is
```

Comment:
Should it be `functionalities`?

```suggestion
  on!) functionalities currently provided by the `srb rbi` command. Tapioca is
```

Reply:
I think both can be correct, I slightly prefer the way that "functionality" reads, as it presents the tool as a holistic approach, rather than a collection of small functionalities.